[Bug Fix] Logging when folder not exist

TYWong99 opened the pull request:
Added GetFile() to file_utils.py

Moved file handler's addHandler() function call after stream handler's, since the creating files would involve usage of logging. ( Would not cause a crash w/o the change, but the console output would be messed up )

Replace all quotation characters: " -> '

TYWong99 committed:
WaterAligner 1.3

Added GetFile() to file_utils.py

Moved file handler's addHandler() function call after stream handler's, since the creating files would involve usage of logging.
( Would not cause a crash w/o the change, but the console output would be messed up )

Replace all quotation characters: " -> '

qtran122 commented on logic/common/file_utils.py:
```diff
@@ -130,4 +131,30 @@ def StripFilename(file_path):
+    # TODO Also read new file and return its content?
+
+    # Can extract the folder's path even if provided with a file instead
+    if filename.endswith("/"):
+        folder_path = filename
+    else:
+        folder_path = "/".join( filename.split("/")[0:-1] )
```
The variable folder_path looks like it only exists within the scope of this if-else statement.

In other languages, this would throw a compiler error. Python allows for this technically, but it's not good coding practice. It's better to declare variables in a way that makes their scope clear to anyone reading the code.

TYWong99 replied:
I usually declare something like `string folder_path;` before the if-statement, I blame it on Python!

qtran122 commented on logic/common/file_utils.py:
```diff
@@ -130,4 +131,30 @@ def StripFilename(file_path):
+
+
+
+def GetFile(filename):
```
'GetFile' doesn't work as a name since this function is not creating a file.

The way I described it in trello, GetFile would have ensured the folder exists as well as create the file - then return the file handler.

But you have that step done outside. Which I think is fine. Can just rename this function to "EnsureFolderExists"

qtran122 reviewed (changes requested):
Almost there!

TYWong99 committed:
Update file_utils.py

Name & Description change.

Declare "folder_path" variable before if-statement instead.

TYWong99 committed:
follow-up

(log_utils)
Wrong function name fixed.

(file_utils)
Replaced log.Info() with print(), and remove import of log_utils to avoid circular imports.

qtran122 reviewed (approved):
LGTM
